src/igvfd/tests/fixtures/schemas/lab.py

```diff
@@ -2,18 +2,21 @@
 
 
 @pytest.fixture
-def other_lab(testapp):
+def other_lab(testapp, pi):
     item = {
-        'title': 'Other lab',
         'name': 'other-lab',
+        'institute_label': 'Stanford',
+        'pi': pi['@id'],
     }
     return testapp.post_json('/lab', item, status=201).json['@graph'][0]
 
 
 @pytest.fixture
-def lab_0_0():
+def lab_0_0(pi):
     return{
         'name': 'Fake Lab',
```

Comment:
I think that the convention that is being suggested above would require a hyphen and lower case for the name. "fake-lab" Unless this was a negative test.